pkg/email/templates/system_intake_lcid_expiration_alert.gohtml

```diff
@@ -0,0 +1,46 @@
+{{template "easi_header.gohtml"}}
+
+<p>The Life Cycle ID that was issued for {{.ProjectName}}{{if .LCIDIssuedAt}} on {{.LCIDIssuedAt}}{{end}} is set to expire on {{.ExpiresAt}}. If your Lifecycle ID expires, your project will be operating under an expired Lifecycle ID and will be added to the Capital Planning Investment Control (CPIC) risk register.</p>
+<p>To avoid this, please email the Governance Team at <a href="mailto:{{.GRTEmail}}">{{.GRTEmail}}</a> within one week to update them with the current status of your project.</p>
+<p>
+  For New IT development projects, please include (if applicable):
```

Comment:
[FIGMA](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=2202-12656&mode=design&t=1W2JkzMlxIJZZAaQ-0) uses different terminology

```suggestion
  For IT systems projects, please include if applicable:
```